Code/Engine/RendererCore/Material/Implementation/MaterialResource.cpp

```diff
@@ -907,6 +912,8 @@ ezMaterialResource::CachedValues* ezMaterialResource::GetOrUpdateCachedValues()
 {
   if (!IsModified())
   {
+    // TODO: reading s_CachedValues outside the Mutex is NOT THREAD-SAFE !!!
```

Comment:
@C-Core you should maybe have a look at this some time

Reply:
I know, I ran into this as well.